<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Saturated Steam Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit_pressure"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_pressure">
         <item>
          <property name="text">
           <string>PSI G</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>BAR G</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>kg/cm2 G</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Kilopascal G</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>7</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit_temperature"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_temperature">
         <item>
          <property name="text">
           <string>Fahrenheit</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Celsius</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Kelvin</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
